distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc

```diff
@@ -0,0 +1,85 @@
+====
+
+* *production* - The production strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required. Each of the backend components is therefore deployed separately. The Agent can be injected as a sidecar on the instrumented application. The Query and Collector services are configured with a supported storage type - currently Elasticsearch. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
+
+* *streaming* - The streaming strategy is designed to augment the production strategy by providing a streaming capability that effectively sits between the Collector and the Elasticsearch backend storage. This provides the benefit of reducing the pressure on the backend storage, under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/using_amq_streams_on_openshift/index[AMQ Streams]/ https://kafka.apache.org/documentation/[Kafka]).
++
+[NOTE]
+====
+The streaming strategy requires an additional Red Hat subscription for AMQ Streams.
+====
+
+[NOTE]
+====
+The streaming deployment strategy is currently unsupported on {ibmzProductName}.
+====
+
+[NOTE]
+====
+There are two ways to install and use {DTProductName}, as part of a service mesh or as a stand alone component. If you have installed {DTShortName} as part of {SMProductName}, you can perform basic configuration as part of the xref:../../service_mesh/v2x/installing-ossm.adoc#installing-ossm[ServiceMeshControlPlane] but for completely control you should configure a Tempo CR and then xref:../../service_mesh/v2x/ossm-observability.html#ossm-config-external-tempo_observability[reference your distributed tracing configuration file in the ServiceMeshControlPlane].
+
+====
+
+include::modules/distr-tracing-tempo-deploy-default.adoc[leveloffset=+1]
+
+[id="customizing-your-tempo-deployment"]
+== Customizing your deployment
+
+ifdef::openshift-enterprise,openshift-dedicated[]
+For information about configuring persistent storage, see xref:../../storage/understanding-persistent-storage.adoc[Understanding persistent storage] and the appropriate configuration topic for your chosen storage option.
+endif::[]
+
+include::modules/distr-tracing-tempo-config-default.adoc[leveloffset=+2]
+
+include::modules/distr-tracing-tempo-config-storage.adoc[leveloffset=+2]
+
+include::modules/distr-tracing-tempo-config-query-frontend.adoc[leveloffset=+2]
+
+== Setting up monitoring for {TempoShortName}
+
+The {TempoOperator} supports monitoring and alerting of each {TempoShortName} component (distributor, ingester, etc.) and exposes upgrade and operational metrics about the operator itself.
```

Comment:
This renders as:

The Tempo Operator supports monitoring and alerting of each distributed tracing platform (Tempo) component (distributor, ingester, etc.) and exposes upgrade and operational metrics about the operator itself.

I think that sounds weird, I'd go with:

The Tempo Operator supports monitoring and alerting of each Tempo component (distributor, ingester, etc.) and exposes upgrade and operational metrics about the operator itself.